io_scene_halo/misc/mattools.py

```diff
@@ -0,0 +1,80 @@
+# ##### BEGIN MIT LICENSE BLOCK #####
+#
+# MIT License
+#
+# Copyright (c) 2021 Steven Garcia
```

Comment:
Change the copyright to say your name or username here. You made this

Reply:
done!